### ISSUE TITLE
GSuite + endpoint verification

### ISSUE THREAD

**Issue by Mistic92** [pomerium/pomerium#1383]
For GCP & Gsuite customers there is addon named Endpoint Verification. Do you think it would be possible to make it working with Pomerium? We are using it with cloud IAP on Loadbalancer but we'd like to move to Pomerium for most services. 

https://cloud.google.com/endpoint-verification/docs/overview
https://chrome.google.com/webstore/detail/endpoint-verification/callobklhcbilhphinckomhgkigmfocg

**Comment by whs** [comment]
I did some investigation last month on this and couldn't find the device state information.

The biggest clue in the documentation I found is that the Cloud IAP auth cookie wraps a Google auth cookie, so you'll be logged out of all Cloud IAP sites. I'm guessing that Cloud IAP use some private API to check whether the wrapped auth cookie is valid, and retrieve the device information from there.

However, I found [Chrome Verified Access](https://developers.google.com/chrome/verified-access/overview) which might be the way to go for device assertion with Chrome.
